airflow/hooks/druid_hook.py

```diff
@@ -53,6 +54,8 @@ def __init__(
         self.max_ingestion_time = max_ingestion_time
         self.header = {'content-type': 'application/json'}
 
+        assert self.timeout >= 1
```

Comment:
I would prefer not to put assert outside of test code. Could you change it to throw exception when timeout is less than 1?

Reply:
Also please use self.assertXXX otherwise we won't see causes.

Reply:
Strike that, missed the location. I would handle this as a ValueError though

Reply:
@awelsh93 
Can you change this line to:
```
if self.timeout < 1:
   raise ValueError("Druid timeout should be equal or greater than 1")
```

Reply:
Done - thanks